src/NuGet.Core/NuGet.ProjectModel/ProjectRestoreMetadata.cs

```diff
@@ -50,6 +50,9 @@ public class ProjectRestoreMetadata : IEquatable<ProjectRestoreMetadata>
         /// </summary>
         public string PackagesPath { get; set; }
 
+        public string CacheFilePath {
+            get { return OutputPath + "project.nuget.json"; }  }
```

Comment:
temp name...Rob suggested project.nuget.cache...the json thing is probably not useful for the user...they shouldn't need to look at this file at all.